src/main/java/monitoringsystemturbo/model/app/Application.java

```diff
@@ -30,14 +35,19 @@ public String getFullPath() {
     }
 
     public Icon findIcon() {
+        if(fullPath.equals("")){
+            return UIManager.getIcon("FileView.directoryIcon");
+        }
         File file = new File(this.fullPath);
         ShellFolder sf = null;
         try {
             sf = ShellFolder.getShellFolder(file);
         } catch (FileNotFoundException e) {
-            e.printStackTrace();
+            //e.printStackTrace();
```

Comment:
it is not the best idea to have empty catch blocks in code. Think what to do with that exception, maybe it should be propagated and shown to user? If it is not such a case or it is not needed, just uncomment this stack trace, imo its better than do nothing